internal/entities/liquidity_provider/trusted_account.go

```diff
@@ -0,0 +1,30 @@
+package liquidity_provider
+
+import (
+	"context"
+	"errors"
+
+	"github.com/rsksmart/liquidity-provider-server/internal/entities"
+)
+
+var (
+	ErrTrustedAccountNotFound = errors.New("trusted account not found")
+	ErrDuplicateAddress       = errors.New("address already exists")
+)
+
+type TrustedAccountDetails struct {
+	Address          string        `json:"address" bson:"address" validate:"required"`
+	Name             string        `json:"name" bson:"name" validate:"required"`
+	Btc_locking_cap  *entities.Wei `json:"btc_locking_cap" bson:"btc_locking_cap" validate:"required"`
+	Rbtc_locking_cap *entities.Wei `json:"rbtc_locking_cap" bson:"rbtc_locking_cap" validate:"required"`
+	Signature        string        `json:"signature" bson:"signature"`
```

Comment:
Signature and hash are not needed to be included in the signed struct, because they are included in the `entities.Signed` generic struct, you can use as example `PeginConfiguration`